attachment_base_synchronize/data/cron.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo noupdate="1">
+        
+    <record model="ir.cron" id="cronjob_run_attachments_metadata">
+        <field name='name'>Run Attachments Metadata</field>
+        <field name='interval_number'>30</field>
+        <field name='interval_type'>minutes</field>
+        <field name="numbercall">-1</field>
+        <field name="active">False</field>
+        <field name="doall" eval="False" />
+        <field name="model_id" ref="model_ir_attachment_metadata"/>
+        <field name="state">code</field>
+        <field name="code">model.run_attachment_metadata_scheduler()</field>
+        <field name="args">([])</field>
```

Comment:
I went through this code and it LGTM. Just remove this `args` line and the Travis build will go green.

Reply:
By Travis I meant Runbot of course ..

Reply:
Let's see if that will do it,